src/testhelp.h

```diff
@@ -41,6 +41,9 @@
 
 #define REDIS_TEST_ACCURATE     (1<<0)
 #define REDIS_TEST_LARGE_MEMORY (1<<1)
+#define REDIS_TEST_BENCHMARK    (1<<2)
+#define REDIS_TEST_FUZZ_CLUSTER (1<<3)
+#define REDIS_TEST_HUGE_KEY     (1<<4)
```

Comment:
how long do all these tests take?
the CI is running all tests doesn't specify these, but it does specify `--accurate`.
maybe it's sufficient to just rely on that, or maybe split one of these tests to be excluded from `raxTest` and into an independent unit that will not be included in the default (`all`) group.